be/src/formats/csv/csv_file_writer.cpp

```diff
@@ -0,0 +1,165 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "csv_file_writer.h"
+
+#include "formats/utils.h"
+#include "output_stream_file.h"
+
+namespace starrocks::formats {
+
+CSVFileWriter::CSVFileWriter(std::string location, std::unique_ptr<csv::OutputStream> output_stream,
+                             const std::vector<std::string>& column_names, const std::vector<TypeDescriptor>& types,
```

Comment:
too many parameters, do you mind put them into a struct?

Reply:
Yes... I prefer this way.